pubsublite/streaming-analytics/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Pub/Sub Lite to Clous Storage",
+  "description": "An Apache Beam streaming pipeline that reads messages from Pub/Sub Lite, applies fixed windowing on the messages, and writes the results to files on Cloud Storage",
+  "parameters": [
+    {
+      "name": "subscription",
+      "label": "Pub/Sub Lite subscription.",
+      "helpText": "Pub/Sub subscription to read from."
+    },
+    {
+      "name": "windowSize",
+      "label": "Window size in minutes",
+      "helpText": "Window size of output files in minutes, default to 1.",
+      "regexes": [
+        "^[1-9][0-9]*"
```

Comment:
Nit: Just like the `^` matches the beginning of the text, `$` matches the end. How about something like `^[0-9]+$`?

Reply:
Done. I needed to do `^[1-9][0-9]=$` to exclude zero.